.babelrc

```diff
@@ -1,5 +1,19 @@
 {
   "presets": [
+    [
+      "env",
```

Comment:
Why the repetition in `env` and `presets`?

Reply:
@Mbarak-Mbigo  the second env is for the tests environment.

Reply:
It works fine without the repetition 👇 
<img width="797" alt="screen shot 2018-01-25 at 12 26 32" src="https://user-images.githubusercontent.com/25595450/35380703-1ed7edd8-01cb-11e8-9437-e308ea635050.png">